Tests/Form/Type/DocumentTypeTest.php

```diff
@@ -43,7 +43,7 @@ public function testDocumentManagerOptionSetsEmOption()
      */
     public function testSettingDocumentManagerAndEmOptionShouldThrowException()
     {
-        $field = $this->factory->createNamed('name', 'document', null, array(
+        $field = $this->factory->createNamed('name', 'Doctrine\Bundle\MongoDBBundle\Form\Type\DocumentType', null, array(
```

Comment:
If you support PHP 5.5+, you should use `DocumentType::class` and import the statement.